### PR TITLE
css-brochure-form

### PR DESCRIPTION
**Pull request by Jason-Humphrey**
Screen Shot 2021-01-22 at 2 55 23 PM

### CODE REVIEW

**Comment by Jason-Humphrey** [comment]
Hopefully this isn't a big mess. I tried to incorporate your changes as suggested. I was still having trouble so i started over. I know you don't like to hear it, but i was up till 5 am working on this. Afraid of falling behind. Which i already am.

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+  .container {
+    position:fixed;
+    size: 100%;
+  }
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:4]: This can be removed.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  h1 {
+    text-align: center;
+    margin-right: 15%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:7]: Woah, that's a lot of margin! What is this doing for you? It should probably be removed.

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  }
+
+  .container {
+    margin: 0 auto;
+    width: 65%;
+    padding: 20px;
+    font-size: 12px;
+  }
+
+  input, select {
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:29]: This shouldn't be indented

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    position:fixed;
+    size: 100%;
+  }
+  h1 {
+    text-align: center;
+    margin-right: 15%;
+    color: #eb6948;
+    font-size: 15px;
+   }
+   body {
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:11]: This shouldn't be indented

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    font-size: 12px;
+  }
+
+  input, select {
+  width: 87%;
+  padding: 12px 20px;
+  margin: 8px 0 8px 0;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  font
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:35]: This needs to be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far, man!! Couple of notes here. A lot have to do with formatting and indentation, make sure you are paying attention to those 👀

File: css-brochure-form/styles.css (hunk 5 above)
Comment (Jason-Humphrey) [reply css-brochure-form/styles.css:35]: all of those lines need to be removed? if i take off border the ugly border comes back 
Screen Shot 2021-01-22 at 7 02 25 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**